extensions/mssql/package.json

```diff
@@ -110,6 +110,14 @@
       {
         "command": "mssqlCluster.livy.cmd.submitFileToSparkJob",
         "title": "%title.submitSparkJob%"
+      },
```

Comment:
It might just make sense to build this into core since this isn't mssql specific

Reply:
This can be done in a separate PR though. Lets get this in soon and send an email to port this to release.

Reply:
I think Karl is not here today, could you approve these changes please?